INSTALL.md

```diff
@@ -119,6 +119,16 @@ export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$(brew --prefix)/opt/libpq/lib/pkgconfi
 export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$(brew --prefix)/opt/openssl@3/lib/pkgconfig"
 export PATH="$(brew --prefix bison)/bin:$PATH"
 ```
+- Install `clang-format-12` 
+> Note: `brew` does not contain a cask for `clang-format-12`.
```

Comment:
Might want to also say that other versions of clang-format will not work properly in the note.